app/signals/apps/classification/admin/admins.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib import admin, messages
+
+
+class TrainingSetAdmin(admin.ModelAdmin):
+    list_display = ('name', 'file', )
+    actions = ["run_training_with_dataset"]
+
+    @admin.action(description="Train model met geselecteerde dataset")
+    def run_training_with_dataset(self, request, queryset):
+        """
+        TODO: run actual training with dataset
+        """
```

Comment:
Feedback van Yuri: Sanity check, checken of alle kolommen geldig zijn. Dataset verifieren. 

Het zou mooi zijn als je feedback op basis van de dataset kan geven.

Reply:
Lijkt me goed om deze sanity checks synchroon uit te voeren, zodat de functioneel beheerder direct feedback terugkrijgt in Django admin.

Als het bestand goed geformatteerd is, dan kun je een asynchrone job triggeren om het model daadwerkelijk te trainen.